Uno.Cupertino/Styles/Application/ColorPalette.xaml

```diff
@@ -12,12 +12,42 @@
 
 		<!-- Light Theme -->
 		<ResourceDictionary x:Key="Light">
```

Comment:
Can you leave a link to the documentation/specs for those RGB values in the comment?

Reply:
added on top of file